docs/concepts/what-is-multi-cluster-replication.md

```diff
@@ -283,14 +292,25 @@ T = 2: replication task from Cluster B arrives in Cluster C, same as above
 
 #### Conflict resolution
 
-When a Workflow Execution History diverges, proper conflict resolution should be applied.
+When a Workflow Execution History diverges, proper conflict resolution is applied.
 
-In Multi-cluster Replication, Workflow Execution History entries (events) are modeled as a tree, as shown in the second example in [Version History](#version-history).
+In Multi-cluster Replication, Workflow Execution History Events are modeled as a tree, as shown in the second example in [Version History](#version-history).
 
 Workflow Execution Histories that diverge will have more than one history branch.
 Among all history branches, the history branch with the highest version is considered the `current branch` and the Workflow Execution's mutable state is a summary of the current branch.
 Whenever there is a switch between Workflow Execution History branches, a complete rebuild of the Workflow Execution's mutable state will occur.
 
+Temporal [Multi-Cluster Replication](/concepts/what-is-multi-cluster-replication) relies on asynchronous replication of Events across Clusters, so in the case of a failover it is possible to have an Activity Task dispatched again to the newly active Cluster due to a replication task lag.
+This also means that whenever a Workflow Execution is updated after a failover by the new Cluster, any previous replication tasks for that Execution cannot be applied.
+This results in loss of some progress made by the Workflow Execution in the previous active Cluster.
+During such conflict resolution, Temporal re-injects any external Events like Signals in the new Event History before discarding replication tasks.
+Even though some progress could rollback during failovers, Temporal provides the guarantee that Workflow Executions won’t get stuck and will continue to make forward progress.
+
+Activity Execution completions are not forwarded across Clusters.
+Any outstanding Activities will eventually timeout based on the configuration.
```

Comment:
```suggestion
Even though some progress could roll back during failovers, Temporal provides the guarantee that Workflow Executions won’t get stuck and will continue to make forward progress.

Activity Execution completions are not forwarded across Clusters.
Any outstanding Activities will eventually time out based on the configuration.
```
Verb forms, so open rather than closed up.